spec/requests/partners_requests_spec.rb

```diff
@@ -124,4 +124,75 @@
       expect(response).to have_http_status(:found)
     end
   end
+
+  describe "PUT #deactivate" do
+    let(:partner) { create(:partner, organization: @organization) }
+
+    context "when the partner successfully deactivates" do
+      it 'performs a PUT request' do
+        expect(partner.status).not_to eq("deactivated")
+        response = double("Response", value: Net::HTTPSuccess)
+        allow(DiaperPartnerClient).to receive(:put).and_return(response)
```

Comment:
I am a little in favor to put this in a `before` block after the context. Since this is "setting it up"
```
    context "when the partner successfully deactivates" do
    before do
        response = double("Response", value: Net::HTTPSuccess)
        allow(DiaperPartnerClient).to receive(:put).and_return(response)
    end 
```